app/src/App/App.tsx

```diff
@@ -24,6 +25,7 @@ function App() {
           <Route path={'/login'} element={<LoginPage/>}/>
           <Route path={'/signup'} element={<SignupPage/>}/>
           <Route path={'/course/:classNum'} element={<CoursePage/>}/>
+          <Route path={'/course/:classNum/rate'} element={<ReviewPage/>}/>
```

Comment:
Maybe change /rate to review or reviews to have the words consistent

Reply:
Thought it should keep with the website name, but makes sense. Changed.